--- a/sources/blowfish/blowfish.tcl
+++ b/sources/blowfish/blowfish.tcl
@@ -15,7 +15,7 @@
 # ----------------------------------------------------------------------
 #
 
-package require Tcl 8.4
+package require Tcl 8.4 9
 package require Itcl
 
 catch {
--- a/sources/blowfish/blowtest.tcl
+++ b/sources/blowfish/blowtest.tcl
@@ -4,7 +4,7 @@
 #
 
 package require Itcl
-package require Tcl 8.4
+package require Tcl 8.4 9
 package require iblowfish
 
 proc h2b {hex} {
--- a/sources/gorilla.tcl
+++ b/sources/gorilla.tcl
@@ -52,12 +52,12 @@ namespace eval ::gorilla {
 # ----------------------------------------------------------------------
 #
 
-if {[catch {package require Tk 8.5} oops]} {
+if {[catch {package require Tk 8.5 9} oops]} {
 	#
-	# Because of using Themed Widgets we need Tk 8.5
+	# Because of using Themed Widgets we need Tk 9.0
 	#
 
-	puts "Password Gorilla has been unable to load Tk 8.5, which is required."
+	puts "Password Gorilla has been unable to load Tk 9.0, which is required."
 	puts "Reason: '$oops'"
 	exit 1
 }
@@ -82,11 +82,11 @@ proc toplevel {path args} {
 
 option add *Dialog.msg.wrapLength 6i
 
-if {[catch {package require Tcl 8.5}]} {
+if {[catch {package require Tcl 8.5 9}]} {
 	wm withdraw .
 	tk_messageBox -type ok -icon error -default ok \
 		-title "Need more recent Tcl/Tk" \
-		-message "The Password Gorilla requires at least Tcl/Tk 8.5\
+		-message "The Password Gorilla requires at least Tcl/Tk 9.0\
 		to run. This smells like Tcl/Tk [info patchlevel].\
 		Please upgrade."
 	exit 1
--- a/sources/isaac.tcl
+++ b/sources/isaac.tcl
@@ -17,7 +17,7 @@
 # ----------------------------------------------------------------------
 #
 
-package require Tcl 8.4
+package require Tcl 8.4 9
 
 namespace eval isaac {
     #
--- a/sources/modules/PWGprogress-1.0.0.tm
+++ b/sources/modules/PWGprogress-1.0.0.tm
@@ -29,7 +29,7 @@
 
 # this makes use of several 8.5 features, so make sure we are running at
 # least 8.5
-package require Tcl 8.5
+package require Tcl 8.5 9
 
 package provide gorillaprogress 1.0.0
 
--- a/sources/pwsafe/pwsafe.tcl
+++ b/sources/pwsafe/pwsafe.tcl
@@ -1,4 +1,4 @@
-package require Tcl 8.4
+package require Tcl 8.4 9
 package require Itcl
 package require sha256
 package require iblowfish
--- a/sources/tcllib/csv/csv.tcl
+++ b/sources/tcllib/csv/csv.tcl
@@ -10,7 +10,7 @@
 # 
 # RCS: @(#) $Id: csv.tcl,v 1.26 2008/10/02 22:26:48 andreas_kupries Exp $
 
-package require Tcl 8.3
+package require Tcl 8.3 9
 package provide csv 0.7.1
 
 namespace eval ::csv {
--- a/sources/tcllib/md5/md5.tcl
+++ b/sources/tcllib/md5/md5.tcl
@@ -24,7 +24,7 @@
 
 # @mdgen EXCLUDE: md5c.tcl
 
-package require Tcl 8.2
+package require Tcl 8.2 9
 namespace eval ::md5 {
 }
 
--- a/sources/tcllib/md5/md5x.tcl
+++ b/sources/tcllib/md5/md5x.tcl
@@ -18,7 +18,7 @@
 #
 # $Id: md5x.tcl,v 1.19 2008/07/04 18:27:00 andreas_kupries Exp $
 
-package require Tcl 8.2;                # tcl minimum version
+package require Tcl 8.2 9;              # tcl minimum version
 
 namespace eval ::md5 {
     variable version 2.0.7
--- a/sources/tcllib/sha1/sha1.tcl
+++ b/sources/tcllib/sha1/sha1.tcl
@@ -25,7 +25,7 @@
 
 # @mdgen EXCLUDE: sha1c.tcl
 
-package require Tcl 8.2;                # tcl minimum version
+package require Tcl 8.2 9;                # tcl minimum version
 
 namespace eval ::sha1 {
     variable  version 2.0.3
--- a/sources/tcllib/sha1/sha1v1.tcl
+++ b/sources/tcllib/sha1/sha1v1.tcl
@@ -25,7 +25,7 @@
 
 # @mdgen EXCLUDE: sha1c.tcl
 
-package require Tcl 8.2;                # tcl minimum version
+package require Tcl 8.2 9;                # tcl minimum version
 
 namespace eval ::sha1 {
     variable version 1.1.0
--- a/sources/tcllib/sha1/sha256.tcl
+++ b/sources/tcllib/sha1/sha256.tcl
@@ -24,7 +24,7 @@
 
 # @mdgen EXCLUDE: sha256c.tcl
 
-package require Tcl 8.2;                # tcl minimum version
+package require Tcl 8.2 9;                # tcl minimum version
 
 namespace eval ::sha2 {
     variable version 1.0.2
--- a/sources/twofish/twofish.tcl
+++ b/sources/twofish/twofish.tcl
@@ -1,4 +1,4 @@
-package require Tcl 8.4
+package require Tcl 8.4 9
 package require Itcl
 
 namespace eval ::itwofish {
